truffle-config.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
     //
     development: {
       host: "127.0.0.1",     // Localhost (default: none)
-      port: 7545,            // Standard Ethereum port (default: none)
+      port: 9545,            // Standard Ethereum port (default: none)
```

Comment:
@victor-yunenko ganache runs on 7545 by default. Let's leave this at the default port?